docs/news.rst

```diff
@@ -2,6 +2,68 @@
 Release Notes
 =============
 
+2.5.0 (2021-10-05)
+==================
+
+Highlights
+----------
+
+- Next major release (3.0) will remove all non-standard single dash long parameter cli parameter
+
+- Added ability generate manifest file from local directory structure
+
+  .. code-block:: bash
+
+        # from the command line
+        synapse manifest ./ --parentId syn123 --manifestFile ./manifest.tsv
+
+- Added ability to generate manifest file from local directory structure
+
+  .. code-block:: bash
+
+        # from the command line
+        synapse manifest ./ --parentId syn123 --manifestFile ./manifest.tsv
+
+- Added ability to describe csv and tsv files stored in Synapse.
+
+  .. code-block:: bash
+
+        # from python
+        import synapseutils
+        synapseutils.describe(syn=syn, entity="syn12345", mode="string")
```

Comment:
@mattfazza I just realized now that there isn't a check for when you put in a random string for mode.  Should there be a check for if a user inputs something other than "string" or "object"?

On another note, should we just leverage the synapse logging system and always return an object.  If the user doesn't want the output printed, they can instantiate the synapse connection with `silent=True`? Thoughts?